### PR TITLE
Update README for @yield and @stack options

### PR DESCRIPTION
**Pull request by chrisdicarlo**
Updated README with examples for new command-line options --with-yields and --with-stacks

### CODE REVIEW

**Comment by chrisdicarlo** [comment]
That's weird, I only have a single commit on this branch with the only file change being the README.md

**Comment by svenluijten** [comment]
Did you branch off of your original branch that was used for #35? That might be causing the issue here 🙂

**Comment by chrisdicarlo** [comment]
Found the issue - forgot to rebase! 😊

File: README.md [hunk 1]
```diff
@@ -92,6 +92,12 @@ $ php artisan make:view index --section=title --section=content
 # Add an inline section to the view
 # Remember to add quotes around the section if you want to use spaces
 $ php artisan make:view index --section="title:Hello world"
+
+# Automatically create sections for each @yield statement in the extended view
+$ php artisan make:view index --extends=app -with-yields
+
+# Automatically create push sections for each @stack statement in the extended view
+$ php artisan make:view index --extends=app -with-stacks
```
Comment (svenluijten) [inline README.md]: These should be `--with-yields` and `--with-stacks`, right?

Comment (chrisdicarlo) [reply README.md]: Oops, yup!

**Comment by svenluijten** [comment]
Thank you!

**Pull request merged**